packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -285,15 +293,15 @@ export class CdpAutomation {
 
       case 'clear:cookie':
```

Comment:
There are a few clear cookie issues logged, does this impact or clean any of those up?

Reply:
No. This just switches Bluebird to be a promise. I did this so to unify our code. The added benefit here is that we get lint failures when we have promises that don't resolve.